modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/js/addCpConfigurationEntry.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {openModal} from 'frontend-js-components-web';
+
+function openAddCPConfigurationEntryModal({
+	addCPConfigurationEntryRenderURL,
+	namespace,
```

Comment:
Read the previously passed data set ID (say, `dataSetId`) and then on whatever action you want to perform, to open the side panel simply

```
Liferay.fire('open-side-panel', { id: dataSetId, ... });
```

which will open the side panel of the data set with id `dataSetId`